### PR TITLE
docs(v2): update steps in the github-actions section

### PR DESCRIPTION
**Pull request by wise-introvert**
Created a warning, asking users to replace email and name in the GitHub action with their email and name.

## Motivation
I ran into issues when setting up github actions.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by wise-introvert** [comment]
I just realised that I didn't update current and older version of the documentation :sweat_smile:. Will do that..

File: website/versioned_docs/version-2.0.0-alpha.73/deployment.mdx [hunk 1]
```diff
@@ -150,6 +150,12 @@ This workflow assumes your documentation resided in `documentation` branch of yo
 1. Copy your private key with `xclip -sel clip < ~/.ssh/id_rsa` and paste a GitHub secret with name `GH_PAGES_DEPLOY`. Copy file content if the command line doesn't work for you. Save your secret.
 1. Create you [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `documentation.yml`.
 
+:::warning
+
+Please make sure that you replace actions@github.com with your github email and gh-actions with your name
```
Comment (HonkingGoose) [inline website/versioned_docs/version-2.0.0-alpha.73/deployment.mdx:155]:
```suggestion
Please make sure that you replace `actions@github.com` with your GitHub email and `gh-actions` with your name.
```

File: website/docs/deployment.mdx [hunk 2]
```diff
@@ -150,6 +150,12 @@ This workflow assumes your documentation resided in `documentation` branch of yo
 1. Copy your private key with `xclip -sel clip < ~/.ssh/id_rsa` and paste a GitHub secret with name `GH_PAGES_DEPLOY`. Copy file content if the command line doesn't work for you. Save your secret.
 1. Create you [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `documentation.yml`.
 
+:::warning
+
+Please make sure that you replace `actions@github.com` with your github email and `gh-actions` with your name
```
Comment (HonkingGoose) [inline website/docs/deployment.mdx:155]:
```suggestion
Please make sure that you replace `actions@github.com` with your GitHub email and `gh-actions` with your name.
```

File: website/versioned_docs/version-2.0.0-alpha.74/deployment.mdx [hunk 3]
```diff
@@ -150,6 +150,12 @@ This workflow assumes your documentation resided in `documentation` branch of yo
 1. Copy your private key with `xclip -sel clip < ~/.ssh/id_rsa` and paste a GitHub secret with name `GH_PAGES_DEPLOY`. Copy file content if the command line doesn't work for you. Save your secret.
 1. Create you [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `documentation.yml`.
 
+:::warning
+
+Please make sure that you replace actions@github.com with your github email and gh-actions with your name
```
Comment (HonkingGoose) [inline website/versioned_docs/version-2.0.0-alpha.74/deployment.mdx:155]:
```suggestion
Please make sure that you replace `actions@github.com` with your GitHub email and `gh-actions` with your name.
```

File: website/versioned_docs/version-2.0.0-alpha.75/deployment.mdx [hunk 4]
```diff
@@ -150,6 +150,12 @@ This workflow assumes your documentation resided in `documentation` branch of yo
 1. Copy your private key with `xclip -sel clip < ~/.ssh/id_rsa` and paste a GitHub secret with name `GH_PAGES_DEPLOY`. Copy file content if the command line doesn't work for you. Save your secret.
 1. Create you [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `documentation.yml`.
 
+:::warning
+
+Please make sure that you replace actions@github.com with your github email and gh-actions with your name
```
Comment (HonkingGoose) [inline website/versioned_docs/version-2.0.0-alpha.75/deployment.mdx:155]:
```suggestion
Please make sure that you replace `actions@github.com` with your GitHub email and `gh-actions` with your name.
```

File: website/versioned_docs/version-2.0.0-beta.0/deployment.mdx [hunk 5]
```diff
@@ -150,6 +150,12 @@ This workflow assumes your documentation resided in `documentation` branch of yo
 1. Copy your private key with `xclip -sel clip < ~/.ssh/id_rsa` and paste a GitHub secret with name `GH_PAGES_DEPLOY`. Copy file content if the command line doesn't work for you. Save your secret.
 1. Create you [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `documentation.yml`.
 
+:::warning
+
+Please make sure that you replace actions@github.com with your github email and gh-actions with your name
```
Comment (HonkingGoose) [inline website/versioned_docs/version-2.0.0-beta.0/deployment.mdx:155]:
```suggestion
Please make sure that you replace `actions@github.com` with your GitHub email and `gh-actions` with your name.
```

Comment (wise-introvert) [reply website/versioned_docs/version-2.0.0-beta.0/deployment.mdx]: Done: [0aae91](https://github.com/facebook/docusaurus/pull/4876/commits/0aae91dfb0aa93770fa843f7356fd0669efe7634)

File: website/versioned_docs/version-2.0.0-alpha.73/deployment.mdx (hunk 1 above)
Comment (wise-introvert) [reply website/versioned_docs/version-2.0.0-alpha.73/deployment.mdx]: Done: [0aae91](https://github.com/facebook/docusaurus/pull/4876/commits/0aae91dfb0aa93770fa843f7356fd0669efe7634)

File: website/docs/deployment.mdx (hunk 2 above)
Comment (wise-introvert) [reply website/docs/deployment.mdx:155]: Done: [0aae91](https://github.com/facebook/docusaurus/pull/4876/commits/0aae91dfb0aa93770fa843f7356fd0669efe7634)

File: website/versioned_docs/version-2.0.0-alpha.74/deployment.mdx (hunk 3 above)
Comment (wise-introvert) [reply website/versioned_docs/version-2.0.0-alpha.74/deployment.mdx]: Done: [0aae91](https://github.com/facebook/docusaurus/pull/4876/commits/0aae91dfb0aa93770fa843f7356fd0669efe7634)

File: website/docs/deployment.mdx (hunk 2 above)
Comment (HonkingGoose) [reply website/docs/deployment.mdx:155]: I think you missed this one, the current file doesn't seem to have this change. 😉

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Comment by slorber** [comment]
thanks

**Pull request merged**